ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -44,9 +45,74 @@ export default class StatusDropdown extends React.Component {
       showNotesModal: false,
       showTagsModal: false,
       selectedValue: this.props.issueTrackers[0].text,
+      jobList: [],
     };
   }
 
+  containsVismet = (title) => {
+    return visualMetrics.find((metric) => title.includes(metric));
+  };
+
+  anyAlertWithVideoResults = (alertSummary) => {
```

Comment:
Looking more into the code added in this module, I realize it's basically doing a JSON enrichment.
We can easily extract this into a separate class & maintain a simpler API for the `StatusDropdown` component.

Reply:
And how would we do that? Do you have an example of such a class in perfherder? This means that we could pass the alertSummary already processed to StatusDowpdown.